regression/tests/helpers/api_clients.py

```diff
@@ -344,56 +345,56 @@ def get_url_from_email(self, matching_string):
         return get_target_url_from_text(matching_string, email_text)
 
 
-class EnrollmentApiClient(object):
+class EdxRestApiBaseClass(object):
     """
-    Enrollment API client
+    Base class for creating EdxRestAPiClient instance
     """
+    api_url_root = None
+    append_slash = True
 
-    def __init__(self, host=None, key=None):
-        self.host = host or ENROLLMENT_API_URL
-        self.key = key or ACCESS_TOKEN
+    def __init__(self):
+        assert self.api_url_root
```

Comment:
nit: remove this assertion

Reply:
hmm, I copied this code from ecommerce as is, I guess the assertion does make some sense here, we don't want to proceed if url is blank